lhotse/bin/modes/recipes/eval2000.py

```diff
@@ -10,12 +10,23 @@
 @prepare.command(context_settings=dict(show_default=True))
 @click.argument("corpus-dir", type=click.Path(exists=True, file_okay=False))
 @click.argument("output-dir", type=click.Path())
+@click.argument(
+    "transcript-dir",
+    type=click.Path(exists=True, file_okay=False),
+    default=None,
+    required=False,
+)
```

Comment:
Should this be an `option` instead of an `argument`, since it is optional?

Reply:
The latest commit should fix this issue. 

Thanks!